packages/mobile/src/screens/product-info/product-info.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FC } from 'react';
+import {
+  ScreenWrapper,
+  View,
+  Text,
+  EyeIcon,
+  ScrollView,
+} from '~/components/components';
+import { selectProductById } from '~/store/products/selectors';
+import { useAppSelector, useCustomTheme, useRoute } from '~/hooks/hooks';
+import { ProductDto, ProductType } from '@vse-bude/shared';
+import { globalStyles } from '~/styles/styles';
+import { formatToDateTime } from '~/helpers/helpers';
+import { RootNavigationParamList } from '~/common/types/types';
+import { RouteProp } from '@react-navigation/native';
+import { Description } from './components/description/description';
+import { ImageCarousel } from './components/image-carousel/image-carousel';
+import { LotPriceBlock } from './components/price-block/lot-price-block';
+import { ProductPriceBlock } from './components/price-block/product-price-block';
+import { SellerInfo } from './components/seller-info/seller-info';
+
+const ProductInfo: FC = () => {
+  const route = useRoute<RouteProp<RootNavigationParamList>>();
+  const { colors } = useCustomTheme();
+  const id = route.params?.id;
+
+  const product = useAppSelector((state) =>
+    selectProductById(state, id as string),
```

Comment:
```suggestion
    selectProductById(state, id),
```

Reply:
as route.params could be undefined - added if statement, with spinner for now for else block